frontend/src/components/Documentation.vue

```diff
@@ -290,9 +290,7 @@
                 ). To clear the RNA levels, select the
                 <i>None</i>
                 option in the drop down list. RNA levels are available for both 2D and 3D Map
-                Viewer. The gene expression levels used are obtained from the file
-                <b>rna_tissue_hpa.tsv.zip</b>
-                in the
+                Viewer. The expression levels used are obtained from the
```

Comment:
With the risk of adding confusion, I would say that what we are looking at for reactions is reaction presence. The number between 0-1 is really a number between 0 and 100, just scaled down. What we did here was to generate 100 bootstraps from the single-cell RNA-Seq data, pool those single-cells into RNA-Seq profiles, and then run ftINIT on each of those, which for each bootstrap determines the presence of each reaction. The number 0-100 represents in how many bootstraps a reaction was present, so the scaled value 0.5 means that the reaction was considered present in 50 out of 100 bootstrap models. Bootstrapping is a way to "randomize" the RNA-Seq data a bit - the purpose of this is to get a metric of how certain we are that a reaction is present. A value of >=0.99 thus represents that we are pretty sure it is present, <= 0.1 that it is not present, and for reactions with values in between we are really not sure if the reaction is present or not. So, the values have very little to do with fluxes. I hope this helps.

Reply:
Similar to [this comment](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1092#discussion_r947634418), it would be nice if we can remove the word `expression` from here as well for now so that we can close this issue. It would be `The levels used are obtained from the ...`.

Reply:
I think the idea behind the original change (before this first suggestion) was to have something generic so it includes genes, reactions, and potentially other types we might have in the future. It seems a bit redundant to have the same sentence again and only switching out `gene` with `reaction`. I might be misunderstanding what you mean though and if that's the case please provide a full suggestion including what should be added for reaction.

Reply:
The original section details how the gene expression data was obtained (from where it was downloaded). An improvement would be to say how it was processed and link to the instructions in the `data-files` repository.

That last sentence, either with `expression levels` but better with `gene expression levels` is correct.

What should be added is a new paragraph - a concise version of Johan's explanation above, something going like this:

> The reaction presence was initially computed as a number between 0 and 100, scaled to 0-1 as per [these instructions](). The number is computed by creating bootstraps from the single-cell RNA-Seq data from the [Human Protein Atlas - Single-cell type Section](), pooling those into RNA-Seq profiles and applying the ftINIT algorithm. A high value >=0.9 means that there is high confidence that the reaction is present, whereas a low value <=0.1 means it is not. For more information on single-cell bootstraps and how ftINIT works, please see [this manuscript]().

Reply:
Thanks @mihai-sysbio. I found the link for the HPA Single-cell stuff. Could you please provide the links to the instructions and the manuscript?

Reply:
The text above has been slightly tweaked and now includes all the links.

Reply:
@johan-gson could you please confirm if the thresholds are: (a) `0.99 and 0.01` or (b) `0.9 and 0.1`? Based on the manuscript I think it's (a).

Reply:
The initial suggestion from this thread as well as the new paragraph has been links have been addressed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1092/commits/63dfa468105b81574bef6c24ef2c07f4c8536825

Reply:
The threshold we used for on is >= 0.99 and off is <= 0.01

Reply:
Suggestions have been made accordingly, so I'm resolving this thread.